patch-lib-analyzer/pom.xml

```diff
@@ -23,9 +23,9 @@
 	<modelVersion>4.0.0</modelVersion>
```

Comment:
Why isn't the maven-reousources-plugin part of this pom as well?

Reply:
Right, even if the JAR is not deployed to Central, it will be contained in the Docker image. I will fix this...

Reply:
Done